### ISSUE TITLE
[Bug]: Automobility Assembler block crashes server upon being loaded in the world

### ISSUE THREAD

**Issue by The-Shortman** [Laskyyy/Create-Astral#330]
### Bug Description

Automobility Assembler block crashes server upon being loaded in the world

### How to Reproduce?

Place the Automobile Assembly Block from Automobility

### Expected Behavior

The game not to crash

### Version

PR #329

### Logs or additional context

https://mclo.gs/F1UZVRC

### Code of Conduct

- [X] I have searched the issue tracker and confirmed that the issue has not been reported
- [X] I have checked the FAQ (if one is present) and my issue is not listed
- [X] I have verified that I am using the latest version of the modpack
- [X] I have verified that I have followed the required steps to set up the modpack

**Comment by The-Shortman** [comment]
Perhaps something that could have been caused by Autos 'n' Rails?

**Comment by Erdragh** [comment]
No, Autos'n'Rails only affects model loading, and isn't even installed on the server:
https://github.com/Laskyyy/Create-Astral/blob/6d660baca2f01ee715e321f6e3b5b8eedf270e42/mods/autosnrails.pw.toml#L1-L3

If you take a look at the modlist in the log you provided, it isn't there

**Comment by The-Shortman** [comment]
This appears to have been resolved and does not crash servers anymore
